index-plants.md

```diff
@@ -5,10 +5,62 @@ subdomain: plants
 gitter: usegalaxy-eu/Lobby
 ---
 
-# Welcome to Galaxy for Plant Biology.
+![Plant Analysis on Galaxy](/assets/media/logo_plants.svg){:.sc-intro-left}
+
+# Welcome to Galaxy for Plant Biology
+{:.no_toc}
+
+The Plant Anaylsis Workbench is a comprehensive set of analysis tools and consolidated workflows. The workbench is based on the [Galaxy framework](https://galaxyproject.org){:target="_blank"},
+which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.
+
+The current version comprises more than XX  bioinformatics tools dedicated to different research areas of single cell biology.
+
+This service is a joint project between different groups from the [XXXXX](http://www.test.ac.uk){:target="_blank"}, the [XXXXX](http://www.test.ac.uk){:target="_blank"}, [XXXXX](http://www.test.ac.uk){:target="_blank"}, and the [University of Freiburg](https://galaxyproject.eu/freiburg/){:target="_blank"}.
+The server is part if the European Galaxy server and is maintained by the [RNA Bioinformatics Center (RBC)](https://www.denbi.de/network/rna-bioinformatics-center-rbc){:target="_blank"} as part of [de.NBI](https://www.denbi.de){:target="_blank"} and [ELIXIR](http://elixir-europe.org){:target="_blank"}.
```

Comment:
@bgruening can you check this?

Reply:
The following should be mentioned:

* dataplant 
* BioDaten 
* VIB? need to ask Frederik
* IPK? need to ask Uwe
* Timo?

Reply:
I added the above partners, but could not find Timo...?

Reply:
https://www.bio.uni-kl.de/organisation/mitarbeiter/m-o/muehlhaus-timo/